Gems/EMotionFX/Code/Source/Editor/Plugins/SkeletonOutliner/JointPropertyWidget.cpp

```diff
@@ -0,0 +1,105 @@
+#include "JointPropertyWidget.h"
```

Comment:
A few of these files are missing the copyright headers at the top, which has failed vaildation:

```
VALIDATOR_FAILED: CopyrightHeaderValidator /home/runner/work/o3de/o3de/Gems/EMotionFX/Code/Source/Editor/Plugins/SkeletonOutliner/JointPropertyWidget.cpp::CopyrightHeaderValidator FAILED - Source file missing O3DE copyright header.
VALIDATOR_FAILED: CopyrightHeaderValidator /home/runner/work/o3de/o3de/Gems/EMotionFX/Code/Source/Editor/Plugins/SkeletonOutliner/JointPropertyWidget.cpp::CopyrightHeaderValidator FAILED - Source file missing any recognized copyrights.
VALIDATOR_FAILED: CopyrightHeaderValidator /home/runner/work/o3de/o3de/Gems/EMotionFX/Code/Source/Editor/Plugins/SkeletonOutliner/JointPropertyWidget.h::CopyrightHeaderValidator FAILED - Source file missing O3DE copyright header.
VALIDATOR_FAILED: CopyrightHeaderValidator /home/runner/work/o3de/o3de/Gems/EMotionFX/Code/Source/Editor/Plugins/SkeletonOutliner/JointPropertyWidget.h::CopyrightHeaderValidator FAILED - Source file missing any recognized copyrights.
```